cloudwatch_logs_common/src/utils/cloudwatch_facade.cpp

```diff
@@ -147,6 +149,37 @@ Aws::CloudWatchLogs::ROSCloudWatchLogsErrors CloudWatchFacade::CreateLogGroup(
   return status;
 }
 
+Aws::CloudWatchLogs::ROSCloudWatchLogsErrors CloudWatchFacade::CheckLogGroupExist(
+  const std::string & log_group)
+{
+  Aws::CloudWatchLogs::ROSCloudWatchLogsErrors status = CW_LOGS_SUCCEEDED;
+
+  Aws::CloudWatchLogs::Model::DescribeLogGroupsRequest describe_log_group_request;
+  describe_log_group_request.SetLogGroupNamePrefix(log_group.c_str());
+
+  const auto & response = this->cw_client_.DescribeLogGroups(describe_log_group_request);
+  if (!response.IsSuccess()) {
+    status = CW_LOGS_LOG_GROUP_NOT_FOUND;
```

Comment:
This case is a request failure, so we cannot assume whether the log group exists or not. So it would be more correct to use `CW_LOGS_FAILED` as the result in this case, and

Reply:
done.